MonoGame.Framework/Platform/Native/Platform.Interop.cs

```diff
@@ -322,6 +334,25 @@ public static partial int Window_ShowMessageBox(
          string[] buttons,
          int count);
 
+    [LibraryImport(MonoGameNativeDLL, EntryPoint = "MGP_Window_GetTextInputState", StringMarshalling = StringMarshalling.Utf8)]
+    [return: MarshalAs(UnmanagedType.U1)]
+    public static partial bool Window_GetTextInputState(MGP_Window* window);
+
+    [LibraryImport(MonoGameNativeDLL, EntryPoint = "MGP_Window_SetTextInputState", StringMarshalling = StringMarshalling.Utf8)]
+    public static partial void Window_SetTextInputState(MGP_Window* window, [MarshalAs(UnmanagedType.U1)] bool state);
+
+    [LibraryImport(MonoGameNativeDLL, EntryPoint = "MGP_Window_GetIMEPosition", StringMarshalling = StringMarshalling.Utf8)]
+    public static partial void Window_GetIMEPosition(MGP_Window* window, ref int x, ref int y, ref int width, ref int height);
```

Comment:
Is this both an in and out parameter?  We have been using `out` for pure output parameters.